File: OperadoresAritmeticos/Arit.c
#include <stdio.h>

int main() {
    int a = 15, b = 4;
    float resultado;
    
    printf("=== OPERADORES ARITMÉTICOS EM C ===\n");
    printf("Valores: a = %d, b = %d\n\n", a, b);
    
    printf("Soma: %d + %d = %d\n", a, b, a + b);
    printf("Subtração: %d - %d = %d\n", a, b, a - b);
    printf("Multiplicação: %d * %d = %d\n", a, b, a * b);
    
    printf("Divisão inteira: %d / %d = %d\n", a, b, a / b);
    resultado = (float)a / b;
    printf("Divisão real: %d / %d = %.2f\n", a, b, resultado);
    
    printf("Módulo: %d %% %d = %d\n", a, b, a % b);
    
    return 0;
}

File: OperadoresAritmeticos/Arit.cs
using System;

class Program {
    static void Main() {
        int a = 25, b = 7;
        double resultado;
        
        Console.WriteLine("=== OPERADORES ARITMÉTICOS EM C# ===");
        Console.WriteLine($"Valores: a = {a}, b = {b}\n");
        
        Console.WriteLine($"Soma: {a} + {b} = {a + b}");
        Console.WriteLine($"Subtração: {a} - {b} = {a - b}");
        Console.WriteLine($"Multiplicação: {a} * {b} = {a * b}");
        
        Console.WriteLine($"Divisão inteira: {a} / {b} = {a / b}");
        resultado = (double)a / b;
        Console.WriteLine($"Divisão real: {a} / {b} = {resultado:F2}");
        
        Console.WriteLine($"Módulo: {a} % {b} = {a % b}");
        
        int x = 10;
        Console.WriteLine($"\nIncremento: x = {x}, ++x = {++x}");
        Console.WriteLine($"Decremento: x = {x}, --x = {--x}");
    }
}